src/app/common/pipes/block-hash-filter.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { BlockHashFilterPipe } from "./block-hash-filter.pipe";
+
+@NgModule({
```

Comment:
I do not see reasons why this pipe and it's brother pipe for event types should be in `common`. Right now it looks as an integral part of the blocks navigation feature.

Reply:
Removed from `common` folder